embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/EmbraceImpl.java

```diff
@@ -272,7 +261,7 @@ void start(@NonNull Context context,
                boolean enableIntegrationTesting,
                @NonNull Embrace.AppFramework appFramework) {
         try {
-            Systrace.startSynchronous("sdk-init");
+            Systrace.startSynchronous("sdk-start");
```

Comment:
Definitely. I was toying with the ideal of adding some of the traces automatically through some wiring in a newly-created modules interface (that can also expose in a generic way what services should be registered), so I was going to do that all at once. But starting with a companion object is good to in case I don't get to this, as this is a side project of sorts of all...